src/applications/gi/containers/ProfilePage.jsx

```diff
@@ -116,10 +116,7 @@ export function ProfilePage({
       name="profilePage"
       className="profile-page vads-u-padding-top--3"
     >
-      <div className="row">
-        {profile.error && <ServiceError />}
-        {!profile.error && content}
-      </div>
```

Comment:
Adding `row` to the `ServiceError` component caused a negative margin to be applied with these nested classes which did not keep the alignment consistent across pages, so `row` is only wrapped around the content.